privacy-security-explainer.md

```diff
@@ -57,9 +60,182 @@ Rounding, quantization, and fuzzing are three categories of mitigations that mod
+* The request originates from a document that is [visible and has focus](#visibility-and-focus)
+* The request originates from a document allowed to use the WebXR [feature policy](#feature-policy) as well as the [underlying sensors' feature policies](#underlying-sensors-feature-policies)
+* User intention is well understood, either via [explicit consent](#explicit-consent) or [implicit consent](#implict-consent)
+
+If these requirements are not met, the promise returned from `requestSession()` must reject.
+
+## Poses
+### XRPose
+When based on sensor data, calls to `XRFrame.getPose()` will expose sensitive information that may be misused in a number of ways, including input sniffing, gaze tracking, or fingerprinting.
+
+Developers indicate the desire for `XRPose` data by calling `XRFrame.getPose()`.
+
+```js
+function onSessionRafCallback(XRFrame frame) {
+    let motionControllerPose = frame.getPose(xrSession.inputSources[0], xrReferenceSpace);
+}
+```
+
+Before returning poses based on sensor data, the UA must ensure that:
+* User intention is well understood, either via [explicit consent](#explicit-consent) or [implicit consent](#implict-consent); alternatively, in cases where the user experience is not negatively affected, quantization may be applied* The request originates from the document which owns the `XRFrame`'s `XRSession`
+* The document is [visible and has focus](#visibility-and-focus)
+* The `XRSession.visibility` is set to `visible` 
+* **TODO** address issues [#696](https://github.com/immersive-web/webxr/issues/696) and [#724](https://github.com/immersive-web/webxr/issues/724)
+
+### XRViewerPose
+The primary difference between `XRViewerPose` and `XRPose` is the inclusion of `XRView` information. More than one view may be present for a number of reasons. One example is a headset which will generally have two views, but may have more to accommodate greater than 180 degree field of views. Another example is a CAVE system. In all cases, when more than one view is present and the physical relationship between these views is configurable by the user, the relationship between these views is considered sensitive information as it can be used to fingerprint or profile the user.
+
+Developers indicate the desire for `XRViewerPose` data by calling `XRFrame.getViewerPose()`.
+
+```js
+function onSessionRafCallback(XRFrame frame) {
+    let viewerPose = frame.getViewerPose(xrReferenceSpace);
+}
+```
+
+As a result, when more than one view is present, the UA must additionally ensure that:
+* User intention is well understood, either via [explicit consent](#explicit-consent) or [implicit consent](#implict-consent)
+* If the device supports configurable or factory-calibrated interpupillary distance that may vary from device to device, then the XRView transform data must be rounded to prevent fingerprinting. Specific precision for rounding is at the discretion of the user agent.
```

Comment:
> factory-calibrated interpupillary distance

Consider rewording? The factory calibration could affect other data such as the view frustum or projection matrices, and in this case the calibrated value would be lens center distance, not interpupillary distance. Also, this is only an issue if these parameters affect the information exposed to WebXR.

Suggestion:

`If view data (view origins, frustum shape, or projection matrices) is affected by configurable or factory-calibrated parameters such as interpupillary distance or variations in screen geometry that may vary from device to device`

Reply:
To piggyback on Klaus' point: if the IPD is statically set by the factory then that's presents a fingerprinting risk, and if it's user calibrated then that presents both a fingerprinting and a profiling risk. I wonder if that's something that can be cleanly expressed here?